src/config.js

```diff
@@ -155,7 +155,7 @@ export const inputKeyToStateKeyMap = {
 export const PACKAGE_FSSPEC = 'fsspec';
 
 export const KEDRO_VIZ_DOCS_URL =
-  'https://docs.kedro.org/projects/kedro-viz/en/latest/';
+  'https://docs.kedro.org/projects/kedro-viz/en/stable/';
```

Comment:
We don't need `/en/stable` at the end I think (unless pointing to specific pages)

```suggestion
  'https://docs.kedro.org/projects/kedro-viz/';
```